requirements/extras.txt

```diff
@@ -5,3 +5,6 @@ pyarrow
 tqdm
 lightning-cloud == 0.5.70 # Must be pinned to ensure compatibility
 google-cloud-storage
+fsspec[s3] # aws s3
```

Comment:
imo, I think it would be great if we could make at least one of them a default install option. (eg: s3)

Reply:
Yes, let's have s3 as the default. As it is the default supported on the Lightning Platform.

Reply:
done!